lib/core/DocsLayout.js

```diff
@@ -97,16 +113,15 @@ class DocsLayout extends React.Component {
                     metadata.localized_id,
                     metadata.next_id
                   )}>
-                  {i18n
-                    ? translation[this.props.metadata.language][
-                        'localized-strings'
-                      ][metadata.next_id] ||
-                      translation[this.props.metadata.language][
-                        'localized-strings'
-                      ].next ||
-                      'Next'
-                    : metadata.next_title || 'Next'}{' '}
-                  →
+                  <span
+                    className={
+                      nextTitle.match(/[a-z][A-Z]/)
+                        ? 'function-name-prevnext'
+                        : 'text-next'
```

Comment:
I couldn't find any `text-next` css class on the new `main.css`.

Reply:
Seems that is the case. Just checking .. It's easy to miss css selector like this :smile: